app/views/search/search.html.erb

```diff
@@ -42,3 +42,6 @@
     </div>
   <% end %>
 </article>
+<script type="application/javascript">
+  window.history.pushState({},"", "<%= url_for params.except(:authenticity_token, :_method, :controller, :action) %>".replace(/&amp;/g, '&'));
```

Comment:
BTW @matthewford params are already in redirect url 
https://github.com/bitzesty/trade-tariff-frontend/blob/master/app/controllers/search_controller.rb#L15
https://github.com/bitzesty/trade-tariff-frontend/blob/master/app/controllers/search_controller.rb#L10